meteor-collection-hooks/index.d.ts

```diff
@@ -0,0 +1,184 @@
+// Type definitions for Meteor package matb33:collection-hooks
+// Project: https://github.com/matb33/meteor-collection-hooks
+// Definitions by: Trygve Wastvedt <https://github.com/twastvedt>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module 'meteor/mongo' {
```

Comment:
Indeed. There should be two declarations (From Meteor@1.3.x onwards): One for the `Mongo` object defined on the global scope, and the second for the `Mongo` object defined in the `meteor/mongo` module.